mmv1/third_party/terraform/services/container/resource_container_node_pool_test.go.tmpl

```diff
@@ -2471,6 +2471,67 @@ resource "google_container_node_pool" "np" {
 `, cluster, networkName, subnetworkName, enableNV, np, enableNV)
 }
 
```

Comment:
TestAccContainerNodePool_withNetworkConfig seems to have a legitimate regression, or may be flaky/newly flaky. Will take a look in more depth when I'm out of a meeting.